analytical_engine/CMakeLists.txt

```diff
@@ -17,10 +17,11 @@ if (POLICY CMP0048)
 endif ()
 project(analytical_engine LANGUAGES C CXX VERSION ${GRAPHSCOPE_ANALYTICAL_VERSION})
 
-option(EXPERIMENTAL_ON "experimental on?" OFF)
+option(NETWORKX "networkx on?" ON)
 option(BUILD_TESTS "Build unit test" ON)
 
-if (EXPERIMENTAL_ON)
+if (NETWORKX)
+    add_definitions(-DNETWORKX)
     add_definitions(-DEXPERIMENTAL_ON)
```

Comment:
This `EXPERIMENTAL_ON` can be deleted?

Reply:
Vineyard v0.2.1 still requires `EXPERIMENTAL_ON`.

Reply:
I will add a TODO comment here